### ISSUE TITLE
No protobuf generated header files

### ISSUE THREAD

**Issue by onehr** [IITH-Compilers/ML-Compiler-Bridge#19]
Hi ml-compiler-bridge developers,

I encountered this issue during the building process:
```
In file included from /home/user/dev/ML-Compiler-Bridge/test/MLBridgeTest.cpp:17:                             
/home/user/dev/ML-Compiler-Bridge/test/include/ProtosInclude.h:9:10: fatal error: grpc/MLBridgeTest_bool/MLBridgeTest_bool.grpc.pb.h: No such file or directory
    9 | #include "grpc/MLBridgeTest_bool/MLBridgeTest_bool.grpc.pb.h"
```

Since I don't know how header files are generated in this project, could you provide some guidelines for solving this issue?
Thanks!

**Comment by svkeerthy** [comment]
Hi, 

Thanks for your interest. Could you please elaborate on the command(s) you used? And, what gave this error?

Best,
Venkat

**Comment by onehr** [comment]
Hi @svkeerthy, 

Here are the commands/steps I have used:
* protobuf v3.13, I just simply follow its README file
* grpc v1.34, I simply just follow the instructions in https://grpc.io/docs/languages/cpp/quickstart/
* create conda environment from mlbridge.yaml
* pip install grpcio-tools
* onnx-runtime, following official document
* ml-compiler-bridge
```bash
# configure ml-compiler-bridge
cmake .. -DCMAKE_BUILD_TYPE=RelWithDebugInfo  -DONNXRUNTIME_ROOTDIR=/home/user/dev/onnxruntime -DTENSORFLOW_AOT_PATH=/home/user/miniconda3/envs/mlbridge/lib/python3.10/site-packages/tensorflow -DCMAKE_EXPORT_COMPILE_COMMANDS=ON
# Building
make -j12
```

Please let me know if you need further information, thanks.

**Comment by svkeerthy** [comment]
Can you please try compiling once after adding `-DPROTOS_DIRECTORY=./test/protos` to the cmake command? We will update the readme and make relevant changes if it works.

**Comment by onehr** [comment]
This cmake variable solved my issue here, thanks!

BTW, I have another question here: what should be the install prefix? ( I got a little bit confused because looks like cmake would install c libraries and Python files all based on that prefix?)

**Comment by svkeerthy** [comment]
Great! 

About Install prefix - Yes. Currently, the C/C++ libraries and the relevant Python files are being moved to the install path based on the prefix that is set. (We are making changes to expose related Python files as a package/whl files :) Hope it helps!

**Comment by onehr** [comment]
Thanks!

(closed by: #20)
